### ISSUE TITLE
"Cannot use 'in' operator to search for 'dialect' in undefined"

### ISSUE THREAD

**Issue by thebspin** [rphlmr/drizzle-lab#30]
When running it i see Peter Griffin instead of my db scheme

<img width="764" alt="image" src="https://github.com/user-attachments/assets/1ef97f21-6b6c-4d02-a4dd-102719f0b4b0">

This is my config:

```js
export default defineConfig({
	dialect: "postgresql",
	schema: "./src/db/schemas.ts",
	out: "./src/db/migrations",
	dbCredentials: {
		url: env.DATABASE_URL,
	},
	migrations: {
		prefix: "timestamp",
	},
});
```

**Comment by rphlmr** [comment]
👋 can you try with `npx drizzle-lab@latest visualizer --debug` and share some metadata it prints?
I have released a new version few minutes ago to fix this issue 👀

**Comment by thebspin** [comment]
Heya sure. (I tried using it via the vscode ext not sure if that makes any difference)

```js
options {
  debug: true,
  config: undefined,
  'save-dir': '.drizzle-lab',
  'project-id': undefined,
  'ts-config': './tsconfig.json',
  port: 64738
}
config {
  dialect: 'postgresql',
  schema: [ '/Users/user/project/src/db/schemas.ts' ],
  out: './src/db/migrations',
  lab: { projectId: 'visualizer' },
  dbCredentials: {
    url: 'postgres://project'
  },
  migrations: { prefix: 'timestamp' },
  projectId: 'visualizer'
}
labEnv {
  PORT: '64738',
  DRIZZLE_LAB_DEBUG: 'true',
  DRIZZLE_LAB_CONFIG_PATH: undefined,
  DRIZZLE_LAB_SAVE_DIR: '.drizzle-lab',
  DRIZZLE_LAB_PROJECT_ID: undefined,
  DRIZZLE_LAB_CWD: '/Users/user/project',
  TS_CONFIG_PATH: './tsconfig.json'
}
```

**Comment by rphlmr** [comment]
Thanks!
I have released a new version with more logs in the VSCode output panel.
![CleanShot 2024-11-17 at 17 31 11@2x](https://github.com/user-attachments/assets/31a83b60-bed3-429b-a887-721671a577ee)

Are you using Cursor? For some reason, the available version is not the latest (0.4.1)
(https://www.cursor.com/how-to-install-extension)

**Comment by thebspin** [comment]
It's working for me now! And no cursor just vscode :)
